backend/legacyapi/plan.go

```diff
@@ -331,7 +335,8 @@ var FeatureMatrix = map[FeatureType][3]bool{
 	FeatureAccessControl:         {false, false, true},
 	FeatureCustomApproval:        {false, false, true},
 	// Collaboration
-	FeatureSharedSQLScript: {false, true, true},
+	FeatureCustomAnnouncement: {false, false, true},
```

Comment:
This should be put under Admin & Security

Reply:
FeatureCustomAnnouncement -> FeatureAnnouncement

Because there are no 2 things "Announcement" and "Custom Announcement"